### PR TITLE
KCO - added info on external payment and checkout

### PR DESCRIPTION
**Pull request by frederikvig**

### CODE REVIEW

File: src/Klarna.Checkout/README.md [hunk 1]
```diff
@@ -318,6 +318,29 @@ When a payment needs an additional review, the payment in EPiServer is set to th
...
+<details>
+  <summary>External Payment Methods & External Checkout (click to expand)</summary>
+  
+Klarna Checkout offers a wide variety of payment methods to cover the main needs of consumers in all markets, which all are included with a simple, single integration.
+ 
+[Here's the full documentation](https://developers.klarna.com/documentation/klarna-checkout/external-payment-methods/) including supported payment and checkouts - we recommend reading through it thoroughly and then coming back here.
+
+![Klarna Checkout External Payment Methods & External Checkouts](https://developers.klarna.com/static/KCO_external-payment-methods.png)
+
+The most important thing to note is that you need to implement the backend integration for the external payment/checkout yourself. So for instance if you wanted to add PayPal you would have to create a redirect URL that has the processing logic for PayPal. Example: klarna.geta.no/processpaypall.
```
Comment (marisks) [inline src/Klarna.Checkout/README.md]:
```suggestion
The most important thing to note is that you need to implement the backend integration for the external payment/checkout yourself. So, for instance, if you wanted to add PayPal, you would have to create a redirect URL that has the processing logic for PayPal. Example: klarna.geta.no/processpaypal.
```
image
image
image

File: src/Klarna.Checkout/README.md [hunk 2]
```diff
@@ -318,6 +318,29 @@ When a payment needs an additional review, the payment in EPiServer is set to th
 The KlarnaPaymentGateway save notes about payment updates at the order.
 ![Order notes](/docs/screenshots/order-notes.PNG?raw=true "Order notes")
 
+</details>
+<details>
+  <summary>External Payment Methods & External Checkout (click to expand)</summary>
+  
+Klarna Checkout offers a wide variety of payment methods to cover the main needs of consumers in all markets, which all are included with a simple, single integration.
```
Comment (marisks) [inline src/Klarna.Checkout/README.md]:
```suggestion
Klarna Checkout offers a wide variety of payment methods to cover the primary needs of consumers in all markets, which all are included with simple, single integration.
```
Suggestions from Gramamrly:
image

image

File: src/Klarna.Checkout/README.md [hunk 3]
```diff
@@ -318,6 +318,29 @@ When a payment needs an additional review, the payment in EPiServer is set to th
...
+ 
+[Here's the full documentation](https://developers.klarna.com/documentation/klarna-checkout/external-payment-methods/) including supported payment and checkouts - we recommend reading through it thoroughly and then coming back here.
```
Comment (marisks) [inline src/Klarna.Checkout/README.md]:
```suggestion
[Here's the full documentation](https://developers.klarna.com/documentation/klarna-checkout/external-payment-methods/), including supported payment and checkouts - we recommend reading through it thoroughly and then coming back here.
```
image

File: src/Klarna.Checkout/README.md (hunk 2 above)
Comment (frederikvig) [reply src/Klarna.Checkout/README.md]: This was copied from Klarna's documentation. I wanted to keep it the same as over there :)

**Pull request merged**